pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mkdocs-multirepo-plugin"
-version = "0.6.3"
+version = "0.6.4"
```

Comment:
Can you please remove this change? I'd like to combine other PRs into one new release. You can keep the `CHANELOG` changes since I can update it after merging this.

Reply:
You'll need to make the change b/c CI automatically creates a new release based on this version number.